### DISCUSSION TITLE
Shortcuts field on main view

### DISCUSSION THREAD

**Discussion by bogdancordos** [netbox-community/netbox#8235] [Ideas]
Hi all,

I can't find a discussion related to this.
It would be useful to have an option to add shortcuts to the main view (homepage), so you can pin a script, a report or any other object that is used more often.
Not sure of how much of a redesign this would involve, so please dismiss if is out of scope.

**Comment by jeremystretch** [comment]
I think it's a neat idea. Also somewhat related to #7855, which proposes displaying a history of recent objects.

We could probably implement a "pin to home"-type button on the view for each object, which would create a record mapping the user account to the object via a generic foreign key. Objects could then be "unpinned" from the main page.

Edit: I've taken the liberty of opening FR #8248 for this.

**Reply by bogdancordos** [reply]
Thank you, Jeremy, for considering this and for opening the FR.
